packages/react-next/src/common/isConformant.ts

```diff
@@ -1,8 +1,9 @@
 import { isConformant as baseIsConformant, IsConformantOptions } from '@fluentui/react-conformance';
 
-export function isConformant(testInfo: IsConformantOptions) {
+export function isConformant(testInfo: Omit<IsConformantOptions, 'componentPath'> & { componentPath?: string }) {
   const defaultOptions = {
     disabledTests: ['has-docblock', 'kebab-aria-attributes'],
```

Comment:
I am confused on why `as-*` tests are included and passing for components in `react-next`. `as` should not be supported on most components here

Reply:
Internally the `as-*` tests check if the component has an `as` prop, and only run if it does.